debian/50-eos.preset

```diff
@@ -15,3 +15,37 @@ disable serial-getty@.service
 disable ssh*.service
 disable ssh.socket
 disable systemd-nspawn@.service
+
+# Disable units masked by Debian, since systemctl preset-all fails to deal with them
```

Comment:
Looks good, but could you add to the comment here saying that this list should be kept in sync with the `/dev/null` links in `debian/systemd.links`.